test/plug/static_test.exs

```diff
@@ -26,6 +27,13 @@ defmodule Plug.StaticTest do
     assert get_resp_header(conn, "content-type")  == ["text/plain"]
   end
 
+  test "serves the file with a custom content type" do
+    conn = call(conn(:get, "/public/fixtures/manifest-file"))
+    assert conn.status == 200
+    assert conn.resp_body == "[]"
+    assert get_resp_header(conn, "content-type")  == ["application/vnd.manifest+json"]
```

Comment:
It's an error everywhere in the file then I believe :) Let's fix it here and leave fixing it everywhere else for some other PR.